.expeditor/scripts/release_habitat/shared.sh

```diff
@@ -22,13 +26,22 @@ import_keys() {
         core
 }
 
-get_latest_pkg_version_in_channel() {
+# Returns the full "release" version in the form of X.Y.Z/DATESTAMP
+get_latest_pkg_release_version_in_release_channel() {
     local pkg_name="${1:?}"
     version=$(curl -s "${HAB_BLDR_URL}/v1/depot/channels/core/$(get_release_channel)/pkgs/${pkg_name}/latest?target=${BUILD_PKG_TARGET}" \
-        | jq -r '.ident | .version')
+        | jq -r '.ident | .version + "/" + .release')
     echo "${version}"
 }
 
+# Returns the semver version in the form of X.Y.Z
+get_latest_pkg_version_in_release_channel() {
+    local pkg_name="${1:?}"
+    local release
+    release=$(get_latest_pkg_release_version_in_release_channel "$pkg_name")
+    echo "$release" | cut -f1 -d"/"
```

Comment:
In an effort to keep the `cut` calls to a minimum, which can get hard to reason about and are prone to breakage if the data going into them changes, can we instead put the retrieval of the "latest" JSON blob from builder in a function, and then have this function call that, piping the results through the more readable 

```sh
jq -r '.ident | .release'
```
?
(This is also subject to the same question above: could we just use a fully-qualified identifier here?)

Reply:
Any thought on this suggestion?